dace/sdfg/graph.py

```diff
@@ -207,15 +207,15 @@ def add_nodes_from(self, node_list):
         for node in node_list:
             self.add_node(node)
 
-    def node_id(self, node):
+    def node_id(self, node: Any) -> int:
```

Comment:
`Any` is no better than no type. Why it is so difficult to accept one concrete type (or union at least)?

Reply:
We can't, because `Graph` is a general data structure.